content/en/network_monitoring/devices/network_topology_map.md

```diff
@@ -15,62 +15,209 @@ further_reading:
 
-{{< img src="/network_device_monitoring/network_topology_map/network_topology_map_search.mp4" alt="The network device topology map, with vendor:cisco added to the search bar, and then the Filter nodes box filtered by nyc. A node is selected and the inspect option chosen, showing its connected nodes. One of the connected nodes is then selected, and the inspect option is selected again, displaying additional connected nodes" video="true" >}}
+{{< img src="/network_device_monitoring/network_topology_map/network_topology_map_search_2.mp4" alt="The network device topology map, with vendor:cisco added to the search bar, and then the Filter nodes box filtered by nyc. A node is selected and the inspect option chosen, showing its connected nodes. One of the connected nodes is then selected, and the inspect option is selected again, displaying additional connected nodes" video="true" >}}
 
 ## Setup
 
-The Datadog Agent version 7.46 and later automatically collects topology data. No additional installation is necessary.
+The Datadog Agent version 7.52 and later automatically collects topology data. No additional installation is necessary.
 
 ### Prerequisites
 
 1. Devices have LLDP (Link Layer Discovery Protocol) and/or CDP (Cisco Discovery Protocol) enabled with SNMP.
-2. Datadog Agent version 7.46 or later is installed.
-3. If you're using [device autodiscovery][3], enable `snmp_listener.collect_topology: true` in the `datadog.yaml` file.
+2. Datadog Agent version 7.52 or later is installed.
+
+## Navigation options
+
+In the Network Topology Map, the following navigation options are available:
+
+### View
+
+1. Under **View By**, select to visualize devices in groups according to tags: 
+
+{{< img src="/network_device_monitoring/network_topology_map/network_view_by_location_2.png" alt="The navigation option, with view by devices and tags selected, highlighting view by location" style="width:80%;" >}}
+
+### Color
+
+2. Under **Color By**, change how nodes on the Device Topology Map are colored based on:
+
+- **Device State**: Display nodes on the Device Topology Map by SNMP reachability.
+- **Ping State**: Display nodes on the Device Topology Map by Ping status. 
+
+{{< img src="/network_device_monitoring/network_topology_map/device_topology_color_by_device.png" alt="The navigation option, with view by color selected, highlighting view by device state" style="width:80%;" >}}
+
+   The following are the definitions of the nodes for each color state:
+   <div style="width:80%; margin: 0 auto;">
+
+   | Color    | Description               |
+   |----------|---------------------------|
+   | Green   | Device is reachable.      |
+   | Red   | Issue with device, such as unreachable through SNMP.  |
+   | Gray    | The device is monitored by NDM, however, no data has been received. For example, if the ping wasn't configured and you opted to `color by` **Ping State** in the Device Topology Map, the devices are displayed in gray. |
```

Comment:
```suggestion
   | Gray    | Device is monitored by NDM; however, no data has been received. For example, if the ping wasn't configured and you opted to `color by` **Ping State** in the Device Topology Map, the devices are displayed in gray. |
```

Reply:
you weren't using articles at the beginning of the other Descriptions, so I removed this one.

I both Ping and ping in the doc. You might want to just use ping, whether it's a noun or a verb. Very common usage.